elasticdl_client/common/k8s_client.py

```diff
@@ -216,6 +378,7 @@ def _create_master_pod_obj(self, **kargs):
             volume=kargs["volume"],
             owner_pod=None,
             env=env,
+            pod_type="master",
```

Comment:
Can we add a enumeration class for the pod_type?

Reply:
Added.